source/_components/binary_sensor.markdown

```diff
@@ -29,5 +29,6 @@ The way these sensors are displayed in the frontend can be modified in the [cust
 - **smoke**: `On` means smoke detected
 - **sound**: `On` means sound detected, `Off` means no sound
 - **vibration**: `On` means vibration detected, `Off` means no vibration
+- **plug**: `On` means device is plugged in, `Off` means device is unplugged
```

Comment:
Please put this line sorted :abcd: in the list.

Reply:
Thanks, I've changed this.